packages/vite/src/node/ssr/__tests__/ssrTransform.spec.ts

```diff
@@ -5,44 +5,45 @@ test('default import', async () => {
   expect(
     (
       await ssrTransform(
-        `import foo from 'vue';console.log(foo.bar)`,
+        `import foo from 'vue';\n` + `console.log(foo.bar)`,
```

Comment:
**question:** Why is such a change needed? I thought they are intended so the code is as small as possible 🤔 
Also I don't see the relation to `"generate more useful names for imported modules"`

Reply:
If you want small code, use a minifier. :)

Reply:
This test case and a couple others were creating confusion by not mimicking idiomatic code, which led to the mistake I corrected [here](https://github.com/vitejs/vite/pull/3952/files#diff-cfc76e75093a8c9e1513ea8238e17e2cfa0fbb5751a0f3d0d10ab157fbfab6f3L50). The fix is tangential to this PR's purpose, but they touch the same function.